WORKSPACE.bzlmod

```diff
@@ -0,0 +1,23 @@
+# Include dependencies which are only needed for development here.
+
+local_repository(
+    name = "mappings_test_external_repo",
+    path = "tests/mappings/external_repo",
+)
+
+# TODO(aiuto): bzlmod chokes on @rules_pkg// in the toolchain labels.
+# Find rpmbuild if it exists.
```

Comment:
Is this a bug in `bzlmod`?

Reply:
It may be a deficiency or a lack of documentation, causing me to deal with it wrong.
I left it as a TODO so that I can establish the core bzlmod file to move us forward, then have a much smaller PR to work with the bzlmod team on.